omero/sysadmins/server-upgrade.txt

```diff
@@ -96,6 +96,16 @@ Since 5.3, all official OMERO.web plugins can be installed from :pypi:`Python Pa
 You should remove all previously installed plugins and install the latest
 versions using `pip <https://pip.pypa.io/en/stable/>`_.
 
+Since 5.3, the `Open with` configuration allows users to open data in other web applications
```

Comment:
Here and below the markup for Open with should be *double* backticks. 

Also, looking at http://www.openmicroscopy.org/site/support/omero5.3-staging/sysadmins/server-upgrade.html#omero-web-configuration, this is the second paragraph in a row which begins "Since 5.3". Could we have "Also introduced in OMERO 5.3, the Open with..." for this sentence?